src/components/feedback/Banner/index.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import styles from './styles.less';
+import Button from '../../general/Button';
+import classNames from 'classnames';
+
+type BannerProps = {
+    message: string;
+    icon?: any;
+    cancelLabel: string;
```

Comment:
The cancel button label should probably be "hard coded" within the component to keep it consistent.